release-tools/make_bump.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -e
+set -x
+script_dir="$(dirname $(realpath -s $0))"
+
+#tools
+project=$1
+tmp_dir=$(mktemp -t -d cc-bump.XXXX)
+release_tool="${tmp_dir}/release-tool"
+hub_bin="${tmp_dir}/hub-bin"
+
+function cleanup {
+	rm  -rf "${tmp_dir}"
+}
+
+trap cleanup EXIT
+
+die()
+{
+	msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+
+
+function build_tools() {
+	pushd ${script_dir}
+	go build -o  "${release_tool}"
+	popd
+	git clone --depth 1 https://github.com/github/hub.git
+	pushd hub
+	./script/build -o "${hub_bin}"
+	popd
+}
+
+function generate_commit() {
+}
+
+
+function bump_project() {
+	${hub_bin} clone "git@github.com:clearcontainers/${project}.git"
+
+	pushd "${project}"
+	${hub_bin} fork --remote-name=fork
+	new_version=$(${release_tool} status --next-bump "${project}")
+	current_version="$(cat VERSION)"
+
+	release_notes_script="./scripts/release_notes.sh"
+	notes_file=notes.md
+	if [[ -x "${release_notes_script}" ]]
+	then
+		${release_notes_script} "${current_version}" "${new_version}" > "${notes_file}"
+	else
+		cat << EOT > ${notes_file}
+# Clear Containers ${new_version}
+
+EOT
+	fi
+
+	echo "${new_version}" > VERSION
+	if [ -f "configure.ac" ] ; then
+		sed -i -e \
+			"s|\(AC_INIT(\[[-a-zA-Z\_]*\],\s*\[\)${current_version}\(\]\)|\1${new_version}\2|g"\
+			configure.ac
+	fi
+	branch="${new_version}-branch"
+	${hub_bin} checkout -b "${branch}" master
+	${hub_bin} add -u
+	${hub_bin} commit -s -m "release: Clear Containers ${new_version}"
```

Comment:
This can use the "Clear Containers" variable. But also, should this be a call to `generate_commit()`?